File: 3rd Semester/OOP_SWE_4302/Lab_9/ConsoleApp1/Person.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ConsoleApp1
{
    public class Person
    {
        public string Name { get; set; }
        public string address { get; set; }
        public int age { get; set; }

        public Person(string Name, string address, int age)
        {
            this.Name = Name;
            this.address = address;
            this.age = age;
        }
    }
}

File: 3rd_Semester/OOP_SWE_4302/comparator/src/person.java
public class person implements Comparable<person> {
    public String name;
    public int age;

    public person(String name, int age) {
        this.name = name;
        this.age = age;
    }

    public int compareTo(person that) {
        if(this.age > that.age) {
            return 1;
        } else if(this.age < that.age) {
            return -1;
        } else {
            return 0;
        }
    }
}
